src/ray/gcs/redis_gcs_client.cc

```diff
@@ -138,14 +148,13 @@ Status RedisGcsClient::Connect(boost::asio::io_service &io_service) {
   resource_table_.reset(new DynamicResourceTable({primary_context_}, this));
 
   actor_accessor_.reset(new RedisActorInfoAccessor(this));
+  job_accessor_.reset(new RedisJobInfoAccessor(this));
+
+  is_connected_ = true;
 
-  Status status = Attach(io_service);
-  is_connected_ = status.ok();
+  RAY_LOG(INFO) << "RedisGcsClient Connected.";
 
-  // TODO(micafan): Synchronously register node and look up existing nodes here
-  // for this client is Raylet.
```

Comment:
This TODO doesn't look fixed?

Reply:
Plan is change, we will not register node in `GcsClient::Connect()`, but call the register method manually.

Reply:
ok